### PR TITLE
Josh webfont

### PR DESCRIPTION
**Pull request by joshuakehrer**
take a look over there

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -109,14 +109,14 @@
           font-variation-settings: "wght" 0;
         }
         to {
-          font-variation-settings: "wght" 1000;
+          font-variation-settings: "wght" 150;
```
Comment (samdiener) [inline docs/index.html:112]: no joosh what have you done? this is so wrong

File: docs/index.html [hunk 2]
```diff
@@ -109,14 +109,14 @@
...
         }
       }
 
       .anim {
         animation-name: animate-fit;
         animation-direction: alternate;
-        animation-duration: 3s;
+        animation-duration: 2s;
```
Comment (samdiener) [inline docs/index.html:119]: also wrong

**Review by samdiener — COMMENTED** [review]
For josh

File: docs/index.html (hunk 1 above)
Comment (joshuakehrer) [reply docs/index.html:112]: this was just an experiment for my font

**Pull request merged**